app/controllers/password_resets_controller.rb

```diff
@@ -0,0 +1,36 @@
+class PasswordResetsController < ApplicationController
+  def new
```

Comment:
You will need to skip authentication / authorisation callbacks for public actions.